lib/ansible/runner.rb

```diff
@@ -218,10 +218,9 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
           fetch_galaxy_roles(playbook_or_role_args)
 
           result = wait_for(pid_file(base_dir)) do
-            AwesomeSpawn.run("ansible-runner", :env => env_vars_hash, :params => params)
+            result = AwesomeSpawn.run!("ansible-runner", :env => env_vars_hash, :params => params)
```

Comment:
@Fryguy you said something (offline) about ansible-runner hard failing and that got me thinking if ansible-runner does fail without starting the subprocess we'll timeout instead of printing the exception.

I replaced my ansible-runner command with:
```
$ cat /usr/local/bin/ansible-runner
#!/bin/bash
echo "Failure!" >&2
exit 1
```

To simulate this and using `AwesomeSpawn.run!` we will raise an exception with the stdout and not wait for the `pid` file.

Now this will raise a: `AwesomeSpawn::CommandResultError (ansible-runner exit code: 1 error was: Failure!)` exception if `ansible-runner` fails.

WDYT?

Reply:
will the nested `result =` conflict?

Reply:
:+1: the outer result isn't needed, removed